doc/Language/js-nutshell.pod6

```diff
@@ -393,9 +393,100 @@ say ++$foo; # 0
 
 =head1 The networking API
 
-=comment TODO
+=head2 Net
 
-# TBD
+In Perl 6, there are two APIs for dealing with networking: C<IO::Socket::INET>
+(for synchronous networking), and  C<IO::Socket::Async> (for asynchronous
+networking).
+
+C<IO::Socket::INET> currently only supports TCP connections. Its API resembles
+that of C's socket API. If you're familiar with that, then it won't take long
+to understand how to use it. For example, here's an example of an echo server
```

Comment:
"example" being used twice in one sentence reads a little odd, you might want to rephrase it to "For example, here's how to create an echo server that closes after receiving one message".